### PR TITLE
SAM-2834; folder icon uses wrong URL in question pool listing

### PR DESCRIPTION
**Pull request by clhedrick**

### CODE REVIEW

**Comment by clhedrick** [comment]
Using .. for a URL in a javascript file is asking for trouble. If there are any changes, its going to get the wrong thing. I've changed all the .. URLs in that file to be relative to the context base. This is a broader change than needed to fix the reported problem. That would have required me to find which of the 7 cases caused the problem and fix just that one. It seems better to fix them all.

File: samigo/samigo-app/src/webapp/js/samigotree.js [hunk 1]
```diff
@@ -148,7 +148,7 @@ function toggleRows(elm) {
  var rows = tables[t].getElementsByTagName("TR");
 
  //var rows = document.getElementsByTagName("TR");
- elm.style.backgroundImage = "url(../../images/folder-closed.gif)";
+ elm.style.backgroundImage = "${pageContext.request.contextPath}/images/folder-closed.gif)";
```
Comment (juanjmerono) [inline samigo/samigo-app/src/webapp/js/samigotree.js]: It seems that you miss "url(" at the beginning.

**Comment by clhedrick** [comment]
I agree. I'll fix it.

**Comment by clhedrick** [comment]
It works to do this in the .js file. I was actually sort of surprised. But the js adds a string to the DOM, and it ends up being processed by JSF.
However there are other issues. I've been unable to test this properly in master, and I just figured out why. Some of the code is triggered by body unload, but that's removed by the inlime coding. This is going to require more work.

**Comment by clhedrick** [comment]
trying again. this has issues

**Comment by alejandrogj** [comment]
@clhedrick 

Just an idea but, maybe we can use the Font awesome icons instead of ".gif".

In the Javascript we would add or remove the css class for the icon

**Pull request closed without merging**